lib/Form/Block/Heading.php

```diff
@@ -82,6 +82,8 @@ private function renderStyle(array $block): string {
     }
     if (!empty($block['params']['line_height'])) {
       $styles[] = 'line-height: ' . $block['params']['line_height'];
+    } elseif (!empty($block['params']['font_size'])) {
+      $styles[] = 'line-height: 1.4;';
```

Comment:
This overrides a value from custom css. I think we should use a different approach.